### PR TITLE
Add rules for good commit messages

### PR DESCRIPTION
**Pull request by VoicuGeo**
Because:
-It's good to have from beginning some rules of
how to write commit messages.
-Good commit messages will help the reviewers to
understand what they have to check

I thought it is necessary to have from beginning
a sets of rules on how to write commit messages.
Adding the rules here everybody will be able to
check them and have them like a guid

This rules should resolve the ticket:
https://github.com/CiprianBucur/Java-project/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,43 @@
...
+3.	Remove unnecessary punctuation marks.
+4.	Use the imperative mood in the subject line.
+Example: “Fix bug” and not “Fixed bug” or “Fixes bug”
+5.	Use the body to explain what changes you have made and why you made them.
+6.	Do not think your code is self-explanatory.
+7.	Do not assume the reviewer understands what the original problem 
+was, ensure you add it.
+8.	Capitalize the subject line and each paragraph.
+9.	Describe the fix, NOT the bug.
+10.	Additionally, add a link to the Jira ticket or supporting information.
```
Comment (CiprianBucur) [inline README.md:23]: I would remove "Jira" since we are not using Jira.

Comment (VoicuGeo) [reply README.md:23]: good point. I will delete it

File: README.md [hunk 2]
```diff
@@ -5,3 +5,43 @@
...
+11.	Follow the commit convention defined by your team.
+
+# Full example:
+Subject: Refactor the coupon UI
+ 
+Body:
+ Because:
+ -The old UI code is fairly slow
```
Comment (CiprianBucur) [inline README.md:31]: Add white space after "-"

File: README.md [hunk 3]
```diff
@@ -5,3 +5,43 @@
...
+ -There were a few unused dependencies
```
Comment (CiprianBucur) [inline README.md:32]: Add white space after "-"

**Comment by CiprianBucur** [comment]
Update steps.java to use the new path of chromedriver.

File: README.md [hunk 4]
```diff
@@ -5,3 +5,43 @@
...
+
+# How to write good commit messages 
+1.	Wrap it to about 72 characters or so.  In some contexts, the first 
+line is treated as the subject of an email and the rest of the text as 
+the body. First line should be followed by a blank line.
+2.	Specify the type of commit.
+3.	Remove unnecessary punctuation marks.
+4.	Use the imperative mood in the subject line.
+    Example: “Fix bug” and not “Fixed bug” or “Fixes bug”
+5.	Use the body to explain what changes you have made and why you made them.
```
Comment (MarinMihaiAlexandru) [inline README.md:17]: If we put a link to the Jira task/ story, I don't think we need to explain why the changes were made. Also, we are 4-5 people working on this, not separate teams, we should have at least a general idea of why a change was made.

Comment (VoicuGeo) [reply README.md:17]: well, most of the time, tickets don't have all the information inside. What we explain is what we've made in the branch and how we implemented. In this way how will come and review it, will know what was out ideas and why we've made like this. This is how I see this point. I don't know what to say but we can ask the team.
